makelib/helm.mk

```diff
@@ -84,6 +88,20 @@ $(HELM_OUTPUT_DIR)/$(1)-$(HELM_CHART_VERSION).tgz: $(HELM_HOME) $(HELM_OUTPUT_DI
 	fi
 	@$(OK) helm package $(1) $(HELM_CHART_VERSION)
 
+helm.generate.$(1): $(HELM_HOME) $(HELM_DOCS)
+	@if [ "$(HELM_DOCS_ENABLED)" != "true" ]; then \
+		$(OK) helm-docs $(1) [disabled]; \
+	else \
+		$(INFO) helm-docs $(1); \
+		$(MAKE) VERSION="master" helm.prepare.$(1); \
+		cd $(HELM_CHARTS_DIR)/$(1) && $(HELM_DOCS); \
```

Comment:
`<chart_directory>/README.md.gotmpl` (`.txt` was only needed to upload it to github as attachment in the other PR) is the default location helm-docs will look for, could be specified with a flag `--template-files=README.md.gotmpl`. The current implementation will look for it in the default location, otherwise will use the default template, we could add a variable to pass in flags something like, `HELM_DOCS_FLAGS`.

Reply:
I see. Good as is then 👍